pkg/util/input.go

```diff
@@ -52,3 +53,21 @@ func AskForConfirmation(message string) bool {
 		" Do you want to continue [y/N]?")
 	return strings.ToUpper(userConfirmation) == "Y"
 }
+
+// Retrive password from OS native credential provider/keychain
+func GetPasswordKeyring(username string) string {
+	password, err := keyring.Get("minishift", username)
+	if err != nil {
+		return ""
+	}
+	return password
+}
+
+// Store the registration password in OS native keychain
+func SetPasswordKeyring(username, password string) error {
+	err := keyring.Set("minishift", username, password)
+	if err != nil {
+		return err
```

Comment:
What if user doesn't have keyring software installed for win or linux, shouldn't this always going to put error? and we are going to ask user to install those?

Reply:
No extra keyring software is required for Windows, for Linux yes, you need to have `gnome-keyring` installed, So if you have a distro with Gnome DE then you already have the keyring software installed.

> shouldn't this always going to put error? 

Yes, that is the idea, if not installed or some other error, and that error is handled by the caller. In our case we just print the error and move on.

Reply:
@anjannath We need to print the error in a way that user can get pointer and take action from the error.